client/app/account/user/security/users/group.service.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+angular.module("UserAccount.services").service("UseraccountGroupsService", class UseraccountGroupsService {
+
+    constructor (OvhHttp) {
+        this.ovhHttp = OvhHttp;
+    }
+
+    getGroups () {
+        return this.ovhHttp.get("/me/identity/group", {
```

Comment:
It seems like endpoints aren't public yet.

Reply:
Will be soon ;-)

Reply:
It's public since yesterday ;)